packages/styles/src/variables/_type.scss

```diff
@@ -135,6 +167,7 @@ $font-curve-big: (
   'xxl': $font-size-small-huge,
 );
 
+// Deprecated
```

Comment:
Why would we deprecate font-curves? Or do you want to create new ones with an updated naming?